Employee.cs

```diff
@@ -0,0 +1,39 @@
+﻿
+namespace DIP
+{
+    public class Employee
+    {
+        private ILogger _logger;
+
+        // property/setter injection
+
+        public ILogger Logger
```

Comment:
remove property injection and use constructor injection.

Reply:
I removed the property injection, constructor injection is already there. I was just showing all the methods